docs/release/3.4/release-3-4-38.md

```diff
@@ -0,0 +1,147 @@
+OSG Software Release 3.4.38
+===========================
+
+**Release Date**: 2019-10-23
+**Supported OS Versions:** EL7, EL6
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   [StashCache-Client 5.5.0](https://github.com/opensciencegrid/StashCache/releases/tag/v5.5.0): Can be installed as a Python package
```

Comment:
This isn't a user-visible change. The previous version we shipped was 5.2.0 and there have been multiple releases since then. How about
```suggestion
-   [StashCache-Client 5.5.0](https://github.com/opensciencegrid/StashCache/releases/tag/v5.5.0): Bugfixes and improved SciTokens support
```